app/controllers/users/verify_account_controller.rb

```diff
@@ -8,9 +8,9 @@ def index
 
     def create
       @verify_account_form = build_verify_account_form
+
       if @verify_account_form.submit
-        flash[:success] = t('account.index.verification.success')
-        redirect_to account_path
+        redirect_to sign_up_completed_url
```

Comment:
Yes, a separate PR is fine. I'll open an issue to discuss how we want to handle this scenario.